core/cmd/app.go

```diff
@@ -506,6 +506,10 @@ func NewApp(client *Client) *cli.App {
 									Name:  "setEnabled",
 									Usage: "enable/disable the key for the given chain",
 								},
+								cli.BoolFlag{
+									Name:  "nuke",
+									Usage: "if set, will delete all current unconfirmed/queued transactions. Use with caution, this can result in nonce gaps or 'stuck' transactions",
```

Comment:
Are fatally errored txes hidden from users? If not, then 'delete' might be confusing, since they are not actually deleted from the DB and can't be deleted from the mempool. Would 'abandon' make sense?